ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/PartitionByParamsFactoryTest.java

```diff
@@ -396,9 +400,35 @@ public void shouldNotChangeValueIfPartitioningByNull() {
     assertThat(result.value, is(GenericRow.fromList(originals)));
   }
 
-  private PartitionByParams partitionBy(final Expression expression) {
+  private PartitionByParams<Struct> partitionBy(final Expression expression) {
+    final ExecutionKeyFactory<Struct> factory = new ExecutionKeyFactory<Struct>() {
```

Comment:
How come we create a new `ExecutionKeyFactory` implementation here rather than using `ExecutionKeyFactory.unwindowed()` with a mock query builder?

Reply:
No good reason, I've changed it to do this.